Nikita.py

```
@@ -353,24 +353,24 @@ def detect2(fake_param=0,initial2=False,d2_srvinfo=""):
+                                                                ]
+            clstr_candidates                                =   [p for p in clstr_candidates if os.path.isfile(p)]
+            if not clstr_candidates:
+                t.debug_print(f"no cluster file found in {str(clstr_dir)}, skipping cluster")
+                continue                                                                                                # пропускаем кластер без реестра, остальные из regs обрабатываем дальше
+            clstr_nonempty                                  =   [p for p in clstr_candidates if os.path.getsize(p) > 0] \
+                                                                or clstr_candidates
+            clstr_file                                      =   max(
+                                                                    clstr_nonempty,
+                                                                    key=lambda p: (os.path.getmtime(p), os.path.getsize(p))
+                                                                )
+            t.debug_print(f"processing {str(clstr_file)} (size={os.path.getsize(clstr_file)})")
```

Ruff (0.15.17)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)